src/mmw/mmw/settings/layer_settings.py

```diff
@@ -233,6 +233,33 @@
         'opacity': 0.618,
         'has_opacity_slider': True,
         'perimeter': pa_perimeter,
+    },
+    {
+        'code': 'drb_catchment_water_quality_tn',
+        'display': 'DRB Catchment Water Quality Data TN Loading Rates',
+        'table_name': 'drb_catchment_water_quality_tn',
+        'vector': True,
+        'overlay': True,
+        'minZoom': 3,
+        'perimeter': drb_perimeter
+    },
+    {
+        'code': 'drb_catchment_water_quality_tp',
+        'display': 'DRB Catchment Water Quality Data TP Loading Rates',
+        'table_name': 'drb_catchment_water_quality_tp',
+        'vector': True,
+        'overlay': True,
+        'minZoom': 3,
+        'perimeter': drb_perimeter
+    },
+    {
+        'code': 'drb_catchment_water_quality_tss',
+        'display': 'DRB Catchment Water Quality Data TSS Loading Rates',
+        'table_name': 'drb_catchment_water_quality_tss',
+        'vector': True,
```

Comment:
On the fence if these should be added as "Boundary" or "Coverage".  My instinct is to say "Coverage" since it isn't just a boundary outline like the others in that category.  It's more similar to the coverages which symbolize values as filled areas.   Not a strong opinion, but worth considering.